srcpkgs/babashka/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'babashka'
+pkgname=babashka
+version=0.0.88
+revision=2
+archs="x86_64"
+build_style=fetch
+hostmakedepends="unzip"
+short_desc="Native fast-starting Clojure scripting environment"
+maintainer="Patrick Pichler <mail@patrickpichler.dev>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/borkdude/babashka"
+distfiles="https://github.com/borkdude/babashka/releases/download/v${version}-${revision}/babashka-${version}-${revision}-linux-static-amd64.zip"
```

Comment:
Please try to build from source. This is just downloading a precompiled binary.

Reply:
Yes, the build is using GraalVM and I'm not sure how to handle it. 

If this is a no-go we you can close this pull request.

Reply:
I'm just a contributor to void so eventually, a member has to decide whether shipping these precompiled binaries would be acceptable but the general policy is of course to try to build from source.

Reply:
Is there an official way on how to get things like this in?

Reply:
Well, the convention then would be to call this package `babashka-bin` (to indicate that this template is fetching precompiled binaries and not building from source) and convince a void pkg-commiter to merge this. But it's very rare void merges 'not-built-from-source' packages.

Reply:
Thank you very much for the information